app/models/reservation.rb

```diff
@@ -18,7 +18,8 @@ class Reservation < ActiveRecord::Base
   validate :matched_item_and_model
   validate :check_status
   validate :status_final_state
-  validate :not_in_past, :available, :check_banned, on: :create
+  validate :not_in_past, :available, :check_banned,
+           :check_consecutive, on: :create
```

Comment:
Shouldn't we check for this when we renew as well? It's interesting - when we renew we have the availability check duplicated in the renewal code (I believe), which seems a little backwards. Since we include the validations module in the Reservation model, don't we have access to the validation methods anywhere in the model methods? I know that this somewhat expands the scope of this issue but it might be worth refactoring how we handle validations.